src/common/components/contactInfo/contactInfo.tpl.html

```diff
@@ -239,7 +238,7 @@ <h4 class="panel-title  border-bottom-small  visible" translate>{{'CONTACT_INFO'
             </div>
           </div>
         </div>
-        <div class="col-sm-6">
+        <div ng-if="$ctrl.useV3 !== 'true'" class="col-sm-6">
```

Comment:
The styling should be updated for v3 to look nicer. I think we should do something about the h4 above in v3 as well. Email doesn't really belong with mailing address, but it feels weird all by itself as well under contact info.

<img width="866" alt="Screenshot 2024-12-18 at 1 58 31 PM" src="https://github.com/user-attachments/assets/abbb0820-f63d-4515-bcb4-6a215e5fbf27" />